### PR TITLE
Codacy fixes for health_helper and system_utils

### PR DESCRIPTION
**Pull request by rahul-hatwar**
Signed-off-by: RAHUL HATWAR <rahulchandrakant.hatwar@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by rahul-hatwar** [comment]
sioned_bucket_versioning_versioned_bucket_32749', 'test_id': 'TEST-32749', 'marks': ['s3_ops']}, {'nodeid': 'tests/s3/versioning/test_put_bucket_versioning.py::TestPutBucketVersioning::test_put_bucket_versioning_enabled_deleted_bucket_33514', 'test_id': 'TEST-33514', 'marks': ['s3_ops']}, {'nodeid': 'tests/s3/versioning/test_versioning_get_head_object.py::TestVersioningGetHeadObject::test_get_head_object_preexisting_32739', 'test_id': 'TEST-32739', 'marks': ['s3_ops']}, {'nodeid': 'tests/s3/versioning/test_versioning_get_head_object.py::TestVersioningGetHeadObject::test_get_head_object_versioned_bucket_32731', 'test_id': 'TEST-32731', 'marks': ['s3_ops']}, {'nodeid': 'tests/s3/versioning/test_versioning_get_head_object.py::TestVersioningGetHeadObject::test_get_head_object_versioned_bucket_32729', 'test_id': 'TEST-32729', 'marks': ['s3_ops']}, {'nodeid': 'tests/s3/versioning/test_versioning_get_head_object.py::TestVersioningGetHeadObject::test_get_head_versioned_object_invalid_32727', 'test_id': 'TEST-32727', 'marks': ['s3_ops']}, {'nodeid': 'tests/s3/versioning/test_versioning_get_head_object.py::TestVersioningGetHeadObject::download_and_check', 'test_id': '', 'marks': []}, {'nodeid': 'tests/s3/versioning/test_versioning_put_object.py::TestVersioningPutObject::test_put_object_preexisting_32724', 'test_id': 'TEST-32724', 'marks': ['s3_ops']}, {'nodeid': 'tests/s3/versioning/test_versioning_put_object.py::TestVersioningPutObject::test_put_object_versioning_enabled_32728', 'test_id': 'TEST-32728', 'marks': ['s3_ops']}, {'nodeid': 'tests/s3/versioning/test_versioning_put_object.py::TestVersioningPutObject::test_put_object_versioning_suspended_32733', 'test_id': 'TEST-32733', 'marks': ['s3_ops']}, {'nodeid': 'tests/security/test_cortx_port_scanner_kubectl_svc.py::test_cortx_port_scanner_kubectl_svc', 'test_id': 'TEST-34217', 'marks': ['security']}, {'nodeid': 'tests/security/test_cortx_port_scanner_netstat.py::test_cortx_port_scanner_netstat', 'test_id': 'TEST-34218', 'marks': ['security', 'skip']}] created at /root/workspace/cortx-test-1/log/te_meta.json
Successfully unmounted directory

======================================================= 2179 tests collected in 21.24s ========================================================

File: commons/utils/system_utils.py [hunk 1]
```diff
@@ -165,7 +151,7 @@ def run_local_cmd(cmd: str = None, flg: bool = False, chk_stderr: bool = False)
     LOGGER.debug("Command: %s", cmd)
     proc = None
     try:
-        proc = Popen(cmd, shell=True, stdout=PIPE, stderr=PIPE)
+        proc = Popen(cmd, shell=False, stdout=PIPE, stderr=PIPE)
```
Comment (ankita-93) [inline commons/utils/system_utils.py:154]: Any reason why this set to False, 
default value is False only but this function is used in many files, Please make sure we are not changing mearning of execution.

File: commons/utils/system_utils.py [hunk 2]
```diff
@@ -526,26 +501,27 @@ def make_dirs(dpath: str, mode: int = None) -> str:
...
+    Make Directory
+    :param pth: Directory path
+    """
     try:
         os.makedirs(pth, exist_ok=True)
-    except OSError as e:
-        if e.errno != errno.EEXIST:
+    except OSError as erroros:
+        if erroros.errno != errno.EEXIST:
             raise
```
Comment (ankita-93) [inline commons/utils/system_utils.py:519]: raise ? what ?

File: commons/helpers/health_helper.py [hunk 3]
```diff
@@ -737,7 +731,7 @@ def get_group_status(crm_mon_res: dict):
         param: crm_mon_res: pcs response from pcs status xml command
         return: dict
         """
-        clone_set = {}
+        clone_set = dict()
         for group in crm_mon_res['group']:
             resource = []
```
Comment (ApurwaMohite) [inline commons/helpers/health_helper.py:736]: list()

**Review by ApurwaMohite — APPROVED** [review]

**Review by seagate-sarang-sawant — APPROVED** [review]

**Pull request merged**